site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -212,14 +212,12 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `helm.installTests`                                      | Add an ability to run `helm test agones` to verify the installation                                                                                                                                                     | `false`                            |
 | `agones.controller.allocationBatchWaitTime`              | Wait time between each allocation batch when performing allocations in controller mode                                                                                                                                  | `500ms`                            |
 
-{{% feature publishVersion="1.26.0" %}}
```

Comment:
This shouldn't be removed. It should be bumped to 1.27.0 and the lines in this table should be merged into the larger table above.